addons/block_code/block_code_plugin.gd

```diff
@@ -91,6 +95,16 @@ func _exit_tree():
 			EditorInterface.set_current_feature_profile("")
 
 
+func _ready():
+	editor_inspector.connect("edited_object_changed", _on_editor_inspector_edited_object_changed)
+
+
+func _on_editor_inspector_edited_object_changed():
+	var block_code: BlockCode = editor_inspector.get_edited_object() as BlockCode
```

Comment:
I was not aware of Godot's behavior to set the casted value to `null` if [doesn't match the casted type](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/static_typing.html#type-casting). That's both neat and frightening.